### PR TITLE
[WIP] Functions.elementary.hyperbolic [GSoD]

### PR DESCRIPTION
**Pull request by Soumi7**
# Updates functions.elementary.hyperbolic to follow latest docstring style guide [GSoD]

#### References to other Issues or PRs

#19591

#### Brief description of what is fixed or changed
- Added examples and parameters section to docstrings in elementary.hyperbolic

#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/functions/elementary/hyperbolic.py [hunk 1]
```diff
@@ -73,7 +73,14 @@ class sinh(HyperbolicFunction):
     r"""
     The hyperbolic sine function, `\frac{e^x - e^{-x}}{2}`.
```
Comment (asmeurer) [inline sympy/functions/elementary/hyperbolic.py:74]: As you're going through, you should fix instances of single backticks to either be dollars for math (like in this case), or double backticks for code.

File: sympy/functions/elementary/hyperbolic.py [hunk 2]
```diff
@@ -73,7 +73,14 @@ class sinh(HyperbolicFunction):
...
 
-    * sinh(x) -> Returns the hyperbolic sine of x
+    * sinh(x) -> Returns the hyperbolic sine of x.
+
+    Examples
+    ========
+
+    >>> from sympy import sinh
+    >>> sinh(1.2)
```
Comment (asmeurer) [inline sympy/functions/elementary/hyperbolic.py:82]: Let's not use floating point numbers for examples. The examples should show that the functions are symbolic.

File: sympy/functions/elementary/hyperbolic.py [hunk 3]
```diff
@@ -1006,6 +1053,18 @@ class asinh(InverseHyperbolicFunction):
...
 
+    Examples
+    ========
+
+    >>> from sympy import asinh
+    >>> from sympy.abc import x
+    >>> asinh(x).diff(x)
+    1/sqrt(x**2 + 1)
+    >>> asinh(1).diff(x)
+    0
```
Comment (asmeurer) [inline sympy/functions/elementary/hyperbolic.py:1064]: This isn't a very instructive example.

File: sympy/functions/elementary/hyperbolic.py [hunk 4]
```diff
@@ -73,7 +73,14 @@ class sinh(HyperbolicFunction):
     r"""
     The hyperbolic sine function, `\frac{e^x - e^{-x}}{2}`.
 
-    * sinh(x) -> Returns the hyperbolic sine of x
+    * sinh(x) -> Returns the hyperbolic sine of x.
```
Comment (asmeurer) [inline sympy/functions/elementary/hyperbolic.py:76]: Let's write these as plain English sentences, like 
```suggestion
    sinh(x) is the hyperbolic sine of x.
```

File: sympy/functions/elementary/hyperbolic.py (hunk 1 above)
Comment (Soumi7) [reply sympy/functions/elementary/hyperbolic.py]: Yes. Will keep this in mind.

File: sympy/functions/elementary/hyperbolic.py (hunk 2 above)
Comment (Soumi7) [reply sympy/functions/elementary/hyperbolic.py]: Changed.

File: sympy/functions/elementary/hyperbolic.py (hunk 3 above)
Comment (Soumi7) [reply sympy/functions/elementary/hyperbolic.py]: Removed the last example.

File: sympy/functions/elementary/hyperbolic.py (hunk 4 above)
Comment (Soumi7) [reply sympy/functions/elementary/hyperbolic.py]: Changed for all the classes.

**Comment by thambi999** [comment]
Fuck don't edit On 25 Sep 2020 07:40, "S.Y. Lee" <notifications@github.com> wrote:
Merged #20084 into master.

**Pull request merged**